websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,169 @@
+			"/shows/recommended": { state: "Recommended shows" },
+			"/shows/anticipated": { state: "Anticipated shows" },
+			"/shows/watched": { state: "The most watched shows" },
+			"/shows/collected": { state: "The most collected shows" },
+			"/movies/trending": { state: "Trending movies" },
+			"/movies/popular": { state: "Popular movies" },
+			"/movies/recommended": { state: "Recommended movies" },
+			"/movies/anticipated": { state: "Anticipated movies" },
+			"/movies/watched": { state: "The most watched movies" },
+			"/movies/collected": { state: "The most collected movies" },
+			"/movies/boxoffice": { state: "Top 10 grossing movies" },
+			"/calendars/my": { details: "Browsing my calendar" },
+			"/calendars/my/shows": {
+				details: "Browsing my calendar",
+				state: "Shows",
+			},
+			"/calendars/my/premieres": {
+				details: "Browsing my calendar",
+				state: "Premieres",
+			},
+			"/calendars/my/new-shows": {
+				details: "Browsing my calendar",
+				state: "New shows",
+			},
+			"/calendars/my/movies": {
+				details: "Browsing my calendar",
+				state: "Movies",
+			},
+			"/calendars/my/dvd": {
+				details: "Browsing my calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/calendars/shows": {
+				details: "Browsing calendar",
+				state: "Shows",
+			},
+			"/calendars/premieres": {
+				details: "Browsing calendar",
+				state: "Premieres",
+			},
+			"/calendars/new-shows": {
+				details: "Browsing calendar",
+				state: "New shows",
+			},
+			"/calendars/movies": {
+				details: "Browsing calendar",
+				state: "Movies",
+			},
+			"/calendars/dvd": {
+				details: "Browsing calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/discover": { details: "Finding new shows/movies" },
+			"/apps": { details: "Browsing apps" },
+			"/vip": { details: "Browsing VIP features" },
+			"/settings": { details: "Browsing settings" },
+			"/auth/signin": { details: "Registering on Trakt" },
+			"/auth/login": { details: "Logging in to Trakt" },
+			"/widgets": { details: "Browsing widgets" },
+		};
+
+	for (const [pathname, data] of Object.entries(pages))
+		if (path.includes(pathname)) presenceData = { ...presenceData, ...data };
+
+	if (path.includes("/users")) {
+		const username = document.querySelector(
+			"#avatar-wrapper >  .emojis-supported > a"
+		).textContent;
+		if (path.includes("/history"))
+			presenceData.details = `Browsing ${username}'s history`;
+		else if (path.includes("/progress"))
+			presenceData.details = `Browsing ${username}'s progress`;
+		else if (path.includes("/collection"))
+			presenceData.details = `Browsing ${username}'s collection`;
+		else if (path.includes("/ratings"))
+			presenceData.details = `Browsing ${username}'s ratings`;
+		else if (path.includes("/lists"))
+			presenceData.details = `Browsing ${username}'s lists`;
+		else if (path.includes("/comments"))
+			presenceData.details = `Browsing ${username}'s comments`;
+		else if (path.includes("/network" || "/network/friends"))
```

Comment:
this is not valid JS

Reply:
you mean `path.includes("/network" || "/network/friends")`?